ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: ipi-conf-azurestack-creds
+  from: upi-installer
+  commands: ipi-conf-azurestack-creds-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: azurestack-cluster-secrets
+    mount_path: /var/run/azurestack-cluster-secrets
+  env:
+  - name: AZURESTACK_ENDPOINT
+    default: 'https://management.ppe3.stackpoc.com'
```

Comment:
Not for this PR, but it would be nice if this information came from the boskos lease rather than hardcoding it.

Reply:
Cool idea. I created https://issues.redhat.com/browse/CORS-1869